src/components/file-input/README.md

```diff
@@ -1 +1,99 @@
-# File input
\ No newline at end of file
+# File input
+
+## When to use
+Use the file input component when the user is required to upload a file from their local computer.
+
+## Adding the Markup
+
+### Wrapper
+
+The file input component is made up of a wrapping `div` with the class `rvt-file` and the data attribute `data-upload-wrapper`.
```

Comment:
I think this could be broken down better. 

> The file input component is made up of a wrapping `div` with the class `rvt-file` and the data attribute `data-upload-wrapper`.

## Adding the markup

The file input component is made up of a wrapper and (list all the other required elements).

### Wrapper

The wrapper for the file input is a `div` with the class `rvt-file` and the data-attribute `data-upload-wrapper`.